docs-ref-services/postgresql-mysql.md

```diff
@@ -142,21 +147,54 @@ pip install azure-mgmt-rdbms
 Please see the [Python SDK authentication](https://docs.microsoft.com/python/azure/python-sdk-azure-authenticate) page for details on obtaining credentials to authenticate with the management client.
```

Comment:
It would be better to tell the user what they need to do or what they need to get from that page to run the following script.  username, password, and how?